products/redhat-jboss-eap.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /redhat-jboss-eap
+-   /redhatjboss
+-   /redhatjbosseap
+
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes#phases
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
+
+releases:
+-   releaseCycle: "7"
+    support: 2023-06-30
+    eol: 2025-06-30
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
+    latestReleaseDate: 2023-03-01
+    link: https://access.redhat.com/articles/6716841
```

Comment:
This page mention the latest update on 2022-11-15, but `latestReleaseDate` is 2023-03-01. Is there a source we could go to check `latestReleaseDate`s ?

Reply:
cf https://access.redhat.com/articles/6999113 
![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/11c846a9-5cad-4f9b-a21d-8814f86aea99)

Reply:
Also https://access.redhat.com/articles/2332721